docs/fsharp/language-reference/results.md

```diff
@@ -42,7 +42,7 @@ type Request =
 
 // Define some logic for what defines a valid name.
 //
-// Generates a Result wich is an Ok if the name valides;
+// Generates a Result which is an Ok if the name valides;
```

Comment:
There's another typo here, @selony; instead of "valides", this presumably was supposed to be "validates". Would you mind making the correction? If it's easier, we can also make it for you.